src/pages/index.js

```diff
@@ -106,9 +178,41 @@ const cardListSection = new Section(
   { items: initialCards, renderer: renderCard },
   ".cards__list"
 );
-cardListSection.renderItems();
+
+function handleCardLike(card) {
+  api
+    .cardLikes(card._cardData._id, card._cardData.isLiked)
```

Comment:
`_cardData`

Please, delete the **low dash** (`underscore`) from methods/fields if you want to use them **outside** a class because the `low dash` shows a **private** field which is used only inside the class.


Please, correct it in `Card` as well